recipe/meta.yaml

```diff
@@ -1,29 +1,28 @@
 {% set name = "depfinder" %}
-{% set version = "2.8.1" %}
+{% set version = "2.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/depfinder-{{ version }}.tar.gz
-  sha256: 5d80bce796ea7a1994fb888ac4056e5ff9d2b390d7d656e75c45eedaef916090
+  sha256: 6b5187d07e0ca913afe88a00141ef098e5372313294a6165bfc6a6b292387894
 build:
   entry_points:
     - depfinder = depfinder.cli:cli
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.6,<3.10
+    - python >=3.6
```

Comment:
Sorry @ericdill, missed this one. I'm so used to just fix the bot's PR with grayskull that I did not pay attention to a human PR. The only issue here is that we need to remove the `>=3.6` but that is already done in #30.